content/v2.0/get-started.md

```diff
@@ -194,6 +195,100 @@ docker exec -it influxdb /bin/bash
 {{% /tab-content %}}
 <!--------------------------------- END Docker -------------------------------->
 
+<!-------------------------------- BEGIN kubernetes---------------------------->
+{{% tab-content %}}
+
+You can install InfluxDB in a local kubernetes environment with Minikube.
+
+1. Install minikube.
+
+    On macOS, run:
+
+    ```
+    brew install minikube
+    ```
+
+2. Start minikube:
+
+    ```
+    minikube start
+    ```
+
+3. Save the following YAML configuration file somewhere on your machine:
```

Comment:
Maybe give them a sample file name and location where to save file?<influxdb-config.yaml>